### PR TITLE
Upgrading XRAY Addon

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/addons/xray.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ const blueprint = blueprints.EksBlueprint.builder()
...
+Once deployed, it allows applications to be instrumented with X-Ray by leveraging the X-Ray SDK.  Examples of such integration can be found on [AWS Docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy_servicelens_CloudWatch_agent_deploy_EKS.html).
 
 ## Functionality
 
-1. Creates the `xray-system` namespace.
-2. Deploys the [`xray-daemon`](https://www.eksworkshop.com/intermediate/245_x-ray/x-ray-daemon/) manifests into the cluster.
-3. Configures Kubernetes service account with IRSA (`AWSXRayDaemonWriteAccess`) for communication between the cluster and the AWS X-Ray service 
+1. Creates the `amazon-cloudwatch` namespace.
+2. Deploys the [`xray-daemon`](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy_servicelens_CloudWatch_agent_deploy_EKS.html) manifests into the cluster.
+3. Configures Node role with policies (`CloudWatchAgentServerPolicy`, `AWSXRayDaemonWriteAccess`) for communication between the cluster and the CloudWatch and X-Ray service.
```
Comment (shapirov103) [inline docs/addons/xray.md:31]: It looks like this setup deploys CW agent and XRay into the cluster. Why would we do that? Can we drop the agent deployment and just deploy XRay? I also assume that CloudWatchAgentServerPolicy won't be needed if we only deploy xray.

**Review by shapirov103 — COMMENTED** [review]
Looks great, I added one comment.

File: lib/addons/xray/xray-ds.yaml [hunk 2]
```diff
@@ -11,111 +11,72 @@
...
+kind: Namespace
 metadata:
-  name: xray-daemon
+  name: amazon-cloudwatch
   labels:
-    app: xray-daemon
+    name: amazon-cloudwatch
+---
+# create role binding for XRay SDK to read config map
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (shapirov103) [inline lib/addons/xray/xray-ds.yaml:23]: Why do we need these roles and role binding if the SA is not even referenced in the ds deployment?

**Review by shapirov103 — COMMENTED** [review]
@elamaran11 getting there, I have one comment.

File: lib/addons/xray/xray-ds.yaml (hunk 2 above)
Comment (elamaran11) [reply lib/addons/xray/xray-ds.yaml]: Makes sense. Removed it. Pleae check now.

**Comment by shapirov103** [comment]
Let's wait for the e2e to pass

**Pull request merged**